src/indra_cogex/sources/pubmed/__init__.py

```diff
@@ -28,24 +29,41 @@
 
 # Settings for downloading content from the PubMed FTP server
 raw_xml = pystow.module("indra", "cogex", "pubmed", "raw_xml")
+# For mapping ISSN to NLM (many-to-one mapping)
+issn_nlm_map_path = resources.join(name="issn_nlm_map.csv.gz")
```

Comment:
is this pre-generated? where's the code that does this?

Reply:
In the pubmed processor

Reply:
Which means we have to ensure the Pubmed processor is run before the wikidata processor. I'll look into that.

Reply:
I would say then it would be good to make the path a constant in the pubmed processor in INDRA that you can import instead of re-define here

Reply:
having a "ensure_issn_nlm_map()` function in the pubmed processor would be the nicest option

Reply:
Oh, I meant in `indra_cogex.sources.pubmed`, but yes, same solution

Reply:
Resolved in https://github.com/bgyori/indra_cogex/pull/139/commits/16eea92ae401924e982400e05ab46d434d569b6d